src/web/js/dashboard/GoogleAPI.js

```diff
@@ -64,26 +74,33 @@ class GoogleAPI {
   getRecentFilesByExt = (ext) => {
     return window.gapi.client.drive.files.list({
       fields: "files(id, name)",
-      q: 'not trashed and fileExtension="' + ext + '"',
+      q: 'not trashed and fileExtension="' + ext + '"'
     });
   }
 
-  getAppDataFileID = (appDataFilename) => {
```

Comment:
There's nothing wrong with the diff. This was intentional. (refer to previous comment on the issue)